### PR TITLE
Remove Profile form from Settings modal

### PR DESCRIPTION
**Pull request by Devorein**
### WHY

[Card](https://app.charmverse.io/charmverse/remove-profile-form-from-settings-modal-006326267268503605)

### CODE REVIEW

File: components/_app/GlobalComponents.tsx [hunk 1]
```diff
@@ -84,6 +85,7 @@ export function GlobalComponents() {
     <>
       <HexagonalAvatarMask id='hexagon-avatar' />
       <MemberProfileDialogGlobal />
+      <EditMemberProfileDialogGlobal />
```
Comment (mattcasey) [inline components/_app/GlobalComponents.tsx:88]: can you make this part of te same modal? when i toggle between them there is a flicker of the backdrop. it's the view/edit of the profile so doesnt seem like we need two modals

**Review by mattcasey — CHANGES_REQUESTED** [review]
can you make this part of the same modal as MemberProfile? when i toggle between them there is a flicker of the backdrop. it's the view/edit of the profile so doesnt seem like we need two modals

File: components/common/PageLayout/components/Sidebar/components/SidebarSubmenu.tsx [hunk 2]
```diff
@@ -153,23 +152,27 @@ export default function SidebarSubmenu({
...
-        >
-          <UserDisplay userId={user?.id} hideName gridColumn='1' gridRow='1/3' />
-          <Typography variant='body2' noWrap>
-            {user?.username}
-          </Typography>
-          <Typography variant='body2' color='secondary'>
-            My Profile
-          </Typography>
-        </MenuItem>
+        {user && (
```
Comment (mattcasey) [inline components/common/PageLayout/components/Sidebar/components/SidebarSubmenu.tsx:155]: why do we have to check if user exists?

Comment (Devorein) [reply components/common/PageLayout/components/Sidebar/components/SidebarSubmenu.tsx:155]: Updated

File: components/members/components/MemberDirectoryGalleryView.tsx [hunk 3]
```diff
@@ -68,7 +68,7 @@ function MemberDirectoryGalleryCard({
 
   function onClickEdit(e: MouseEvent<HTMLElement>) {
     e.stopPropagation();
-    openSettings('profile');
+    setIsOpen(true);
```
[the remark this answers is not in the document]
Comment (Devorein) [reply components/members/components/MemberDirectoryGalleryView.tsx:71]: Good point, updated in the refactor

**Pull request merged**